medio/backends/itk_io.py

```diff
@@ -49,26 +58,43 @@ def read_img(input_path, desired_axcodes=None, header=False, components_axis=Non
         if (desired_axcodes is None) or (desired_axcodes == metadata.ornt):
             image_np = ItkIO.itk_img_to_array(img)
         else:
-            orig_ornt = metadata.ornt  # store the original orientation before the reorientation
+            orig_ornt = (
+                metadata.ornt
+            )  # store the original orientation before the reorientation
```

Comment:
```suggestion
            # store the original orientation before the reorientation
            orig_ornt = metadata.ornt
```

move comments to the line above.
In fact, this is unnecessary comment, we can infer it by the variable name :)